datafusion/sqllogictest/test_files/union.slt

```diff
@@ -540,8 +540,8 @@ UnionExec
 ----AggregateExec: mode=FinalPartitioned, gby=[Int64(1)@0 as Int64(1)], aggr=[]
 ------CoalesceBatchesExec: target_batch_size=8192
 --------RepartitionExec: partitioning=Hash([Int64(1)@0], 4), input_partitions=4
-----------AggregateExec: mode=Partial, gby=[1 as Int64(1)], aggr=[]
-------------RepartitionExec: partitioning=RoundRobinBatch(4), input_partitions=1
+----------RepartitionExec: partitioning=RoundRobinBatch(4), input_partitions=1
```

Comment:
Why did these plans change? Given the inputs have a single input row, I don't think it actually matters if the `AggregateExec` is before or after the `RepartitionExec`

Reply:
We have talked about it was okay, but I can't exactly remember how it was. @mustafasrepo is more familiar with that topic and can provide a clear explanation of the underlying reason.

Reply:
Actually @alamb you are right. Previous and current plan has no difference, as long as there is repartition in it. Plan shouldn't contain Repartition in the first place. (Since we have single row at the input.) I will file a fix for that.